core/src/raw/http_util/client.rs

```diff
@@ -20,21 +20,26 @@ use std::fmt::Formatter;
 use std::future;
 use std::mem;
 use std::str::FromStr;
+use std::sync::Arc;
 
 use futures::TryStreamExt;
 use http::Request;
 use http::Response;
+use once_cell::sync::Lazy;
 use raw::oio::Read;
 
 use super::parse_content_encoding;
 use super::parse_content_length;
 use super::HttpBody;
 use crate::*;
 
+/// Http client used across opendal for loading credentials.
+pub static CREDENTIAL_CLIENT: Lazy<reqwest::Client> = Lazy::new(reqwest::Client::new);
```

Comment:
How about naming it `GLOBAL_REQWEST_CLIENT`? 

Please add in comments that this is merely a temporary solution because `reqsign` requires a `reqwest` client to be passed. We will remove it after the next major version of `reqsign`, which will enable users to provide their own client.